.github/release.yml

```diff
@@ -0,0 +1,26 @@
+# This file has been mostly taken verbatim from https://github.com/pymc-devs/pymc/blob/main/.github/release.yml
+#
+# This file contains configuration for the automatic generation of release notes in GitHub.
+# It's not perfect, but it makes it a little less laborious to write informative release notes.
+# Also see https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
+changelog:
+  exclude:
+    labels:
+      - no releasenotes
+  categories:
+    - title: Major Changes 🛠
+      labels:
+        - major
+    - title: New Features & Bugfixes 🎉
+      labels:
+        - bug
```

Comment:
The only difference between this file and the one above is the "bug" label rather than "defects".